src/msg/Message.cc

```diff
@@ -79,7 +79,7 @@ using namespace std;
 #include "messages/MOSDPGTrim.h"
 #include "messages/MOSDPGMissing.h"
 #include "messages/MOSDScrub.h"
-#include "messages/MOSDRepScrub.h"
+#include "messages/MOSDBuildScrubMap.h"
```

Comment:
I'm not sure we want to change the name (if I suggested it, I am sorry!).  For debugging purposes, it would be nice if two osds in a mixed cluster agree on the name for the same message.